ads/navegg.md

```diff
@@ -0,0 +1,43 @@
+<!---
+Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Navegg
+
+If you want to serve ads with Navegg data you need to use this adapter.
+
+The Navegg adapter only supports Doubleclick for now.
+For the most up-to-date list of Doubleclick supported parameters and usage please refer to Doubleclick reference guide [here](google/doubleclick.md).
```

Comment:
I would move this line about supported parameters to Configuration below. And then specifically indicate that `data-acc` is a required param. while all Doubleclick params are supported.